.github/workflows/build.yml

```diff
@@ -28,30 +28,10 @@ jobs:
         uses: ocaml/setup-ocaml@v2
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          dune-cache: ${{ matrix.os != 'macos-latest' }}
 
-      - name: Install Linux system packages
-        if: runner.os == 'Linux'
-        run: sudo apt-get -y update && sudo apt-get -y install libffi-dev libgmp-dev libpcre3-dev ncurses-dev pkg-config tzdata pandoc
-
-      - name: Install macOS system packages
-        if: runner.os == 'macOS'
-        run: brew install gmp libffi pcre pkg-config pandoc
```

Comment:
Since Linuxbrew is installed on GitHub Actions by default, you can do it in one step